src/shared/proto/vision.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+import "shared/proto/geometry.proto";
+
+message RobotState
+{
+    Point global_position         = 1;
```

Comment:
Should add units to everything in this file (but _thank you_ for specifying global)

Reply:
added